package.json

```
@@ -13,7 +13,7 @@
 	},
 	"scripts": {
 		"setup:tools": "yarn && composer install && TEXTDOMAIN=wporg composer exec update-configs",
-		"setup:wp": "wp-env run cli \"bash env/setup.sh\"",
+		"setup:wp": "wp-env run cli \"bash env/setup.sh\" && ./env/refresh.sh",
```

You can just call the step as well:

```suggestion
		"setup:wp": "wp-env run cli \"bash env/setup.sh\" && yarn setup:refresh",
```